docs/contrib/DatabaseMath.md

```diff
@@ -78,9 +95,11 @@ int x_snap = (x / pitch) * pitch;
 
 The use of rounding in existing code that uses floating-point
 representations is to compensate for the inability to represent floating-point
-fractions exactly. Results like 5.99999999992 need to be "fixed".
+fractions exactly. Results like $5.99999999992$ need to be "fixed".
 This problem does not exist if fixed-point arithmetic is used.
 
+### Conclusion
+
```

Comment:
This is the most important guideline for coding distances that should be front and center.